huaweicloud/services/cbh/resource_huaweicloud_cbh_instance.go

```diff
@@ -214,6 +214,16 @@ func resourceCBHInstanceCreate(ctx context.Context, d *schema.ResourceData, meta
 		return diag.Errorf("error waiting for CBH instance (%s) creation to active: %s", d.Id(), err)
 	}
 
+	// After successfully creating an instance using the first security group ID, check if it is necessary to update the
+	// security group to the target value.
+	securityGroupIDs := d.Get("security_group_id").(string)
+	sgIDList := strings.Split(securityGroupIDs, ",")
+	if len(sgIDList) > 1 {
+		if err := updateSecurityGroup(client, d); err != nil {
```

Comment:
```suggestion
	securityGroupIDs := d.Get("security_group_id").(string)
	sgIDs := strings.Split(securityGroupIDs, ",")
	if len(sgIDList) > 1 {
		if err := updateSecurityGroup(client, d.Id, sgIDs); err != nil {
```
the definition `func updateSecurityGroup(client *golangsdk.ServiceClient, resourceId string, sgIDs []string) error {}` be better

Reply:
done.